engine/src/main/java/org/pentaho/di/kitchen/KitchenCommandExecutor.java

```diff
@@ -126,7 +128,7 @@ public Result execute( Params params ) throws Throwable {
         if ( job == null ) {
 
           // Try to load the job from file, even if it failed to load from the repository
-          job = loadJobFromFilesystem( params.getLocalInitialDir(), params.getLocalFile() );
+          job = loadJobFromFilesystem( params.getLocalInitialDir(), params.getLocalFile(), params.getBase64Zip() );
```

Comment:
Wondering if we still need to use the getLocalFile() or can we use the inputFile in which case we may not need to send this parameter from PentahoServer